posts/jhipster-5-release.md

```diff
@@ -0,0 +1,196 @@
+
+In JHipster 5, the team has taken this even further, and you can now fully describe your application options and generate it based on the JDL. To celebrate this new version, JHipster has open-sourced an online editor to write your JDL document and view UML like graphs of the entities called the [JDL Studio](https://start.jhipster.tech/jdl-studio/)!
+
+In order to generate the application via the import-jdl sub-generator, we simply create out JDL files with extra flags dictating the answers to questions as seen in use by JHipster in the past. Here is a simple sample of a JDL file to create three applications, and some entities for them. To run it, save the jdl file, and run `jhipster import-jdl <jdl-file>`
+
+```groovy
+application {
+  config {
+    baseName myFirstApp,
+    databaseType postgresql,
+    applicationType monolith,
+    packageName com.mycompany.myfirstapp,
+  }
+  entities * except C, D
+}
+
+application {
+  config {
+    baseName mySecondApp,
+    databaseType postgresql,
+    applicationType microservice,
+    serverPort 8091,
+  }
+  entities E
+}
+
+application {
+  config {
+    baseName myThirdApp,
+    databaseType postgresql,
+    applicationType microservice,
+    serverPort 8092,
+  }
+  entities F
+}
+
+entity A
+entity B
+entity C
+entity D
+entity E
+entity F
+
+microservice E with mySecondApp
+```
+
+After running `import-jdl` on this file, you will be informed to manually run JHipster on the corresponding folders it creates. Navigate to each directory, and run the `jhpster`. This will create the application as specified in the JDL file, and once completed, commit with git.
+
+See [the JDL documentation](https://www.jhipster.tech/jdl/) for more about JDL.
+
+
+# Conclusion
+
+With this new version, the JHipster team proves again they follow the cutting edge trends of the industry (React, Spring boot 2, Kotlin). The opinionated generator is still the best option to get started quickly with a new project using the best practices.
+
+To celebrate this new release, the first [JHipster conference](https://blog.ippon.tech/jhipster-conf-2018) was held in Paris last week and the videos will be published soon.
+
+If you are new to JHipster, now is a perfect time to start using it. Join the community, help contribute by following [the guide](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md), 
+
+Ippon is proud to use and support JHipster to offer quick, quality solutions to our clients. If you need help getting started with JHipster or migrate to the new latest version, please [reach out to us](https://blog.ippon.tech/contact/)!
```

Comment:
Ippon is proud / to our clients -> goes from the third person to the first person